src/test/java/racingcar/repository/RacingGameRepositoryImplTest.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import racingcar.repository.mapper.RacingGameInfo;
+
+@SpringBootTest
+class RacingGameRepositoryImplTest {
+
+    @Autowired
+    RacingGameRepositoryImpl repository;
+
+    @DisplayName("게임 정보 저장 후 조회")
+    @Test
+    void save() {
+        // given
+        final String winners = "저문,헤나";
+        final int trial = 10;
+
+        // when
+        final int savedId = repository.save(winners, trial);
+        final Optional<RacingGameInfo> maybeRacingGameInfo = repository.findById(savedId);
+
+        assertTrue(maybeRacingGameInfo.isPresent());
+
+        final RacingGameInfo racingGameInfo = maybeRacingGameInfo.get();
+
+        // then
+        assertThat(racingGameInfo)
+                .hasFieldOrPropertyWithValue("id", savedId)
+                .hasFieldOrPropertyWithValue("winners", winners)
+                .hasFieldOrPropertyWithValue("trial", trial);
```

Comment:
검증을 위한 값은 특별한 상황이 아니면 하드코딩 해주는 것이 좋아요!

아래 글을 참고해보면 좋겠어요 😄 
[검증부 (assert / expect)는 하드코딩한다](https://jojoldu.tistory.com/615)

Reply:
와우 감사합니다 😄

다음과 같이 수정했습니다.

```suggestion
                .hasFieldOrPropertyWithValue("winners", "저문,헤나")
                .hasFieldOrPropertyWithValue("trial", 10);
```

하드 코딩이 아니었을 때 다음과 같은 문제가 발생할 수 있다는 것을 알게 됐습니다 !

1. 프로덕션 코드와 테스트가 강결합 될 수 있다.
2. 도메인 로직 그 자체를 테스트 코드 안에 복사하여 붙여넣으면 동일한 버그가 나오기에 오류인지 아닌지 판단하기 어렵다.
3. 구현부를 테스트에서 먼저 작성할 가능성이 있다.